grafana-plugin/src/components/RenderConditionally/RenderConditionally.tsx

```diff
@@ -2,9 +2,20 @@ import React, { FC, ReactNode } from 'react';
 
 interface RenderConditionallyProps {
   shouldRender?: boolean;
-  children: ReactNode;
+  children?: ReactNode;
+  render?: () => ReactNode;
   backupChildren?: ReactNode;
 }
 
-export const RenderConditionally: FC<RenderConditionallyProps> = ({ shouldRender, children, backupChildren = null }) =>
-  shouldRender ? <>{children}</> : <>{backupChildren}</>;
+export const RenderConditionally: FC<RenderConditionallyProps> = ({
+  shouldRender,
+  children,
+  render,
+  backupChildren = null,
+}) => {
+  if (render) {
+    return shouldRender ? <>{render()}</> : null;
```

Comment:
just thinking if in this case we shouldn't use backupChildren instead of null here